docs/guide/08-tools.md

```diff
@@ -125,79 +125,74 @@ Build tools take your code and make it production-ready. Among the things you ma
-  "shell": "src/myapp.js",
-  "sources": [
-    "src/**.js",
-    "manifest/**",
-    "package.json"
-  ],
-  "extraDependencies": [
-    "node_modules/@webcomponents/webcomponentsjs/bundles/**"
-  ],
-  "builds": [
-    {"preset": "es6-bundled"}
-  ]
-}
-```
-
-This configuration specifies that the app has an HTML entrypoint called `index.html`, has a main JavaScript file (app shell) called `src/myapp.js`. It will produce a single build, bundled but not transpiled to ES5. For details on the polymer.json file, see [polymer.json specification](https://polymer-library.polymer-project.org/3.0/docs/tools/polymer-json) on the Polymer library site.
-
-To build the project, run the following command in your project folder:
-
-`polymer build`
-
-For more on building with Polymer CLI, see [Build for production](https://polymer-library.polymer-project.org/3.0/docs/apps/build-for-production) in the Polymer library docs.
+open-wc also has [Rollup build resources](https://open-wc.org/building/building-rollup.html). 
 
 ### Build your project with webpack
 
+webpack is a powerful build tool with a large ecosystem of plugins. 
 
-See the Open Web Components default webpack configuration provides a great starting point for building projects that use lit-html. See their [webpack page](https://open-wc.org/building/building-webpack.html#default-configuration) for instructions on getting started. 
-
-### Build considerations for other tools {#build-considerations}
+See the open-wc default webpack configuration provides a great starting point for building projects that use lit-html. See their [webpack page](https://github.com/open-wc/open-wc/tree/master/packages/building-webpack) for instructions on getting started. 
 
+### Build considerations  {#build-considerations}
 
 If you're creating your own configuration for webpack, Rollup, or another tool, here are some factors to consider:
 
-* ES6 to ES5 transpilation.
+* ES6 to ES5 compilation.
 * Transforming JavaScript modules to other formats for legacy browsers.
 * lit-html template minification.
+* Polyfills.
+
+For more details on these considerations, see the LitElement [Build for production](https://lit-element.polymer-project.org/guide/build) guide. lit-html has the same requirements as LitElement, except that lit-html requires only the [Template polyfill](#template-polyfill), not the full Web Components polyfills.
 
-#### Transpilation and module transform
+#### Compilation and module transform {#transpilation-and-module-transform}
 
-You build tools need to transpile ES6 features to ES5 for legacy browsers. 
+You build tools need to compile ES6 features to ES5 for legacy browsers. 
 
 If you're working in TypeScript, the TypeScript compiler can generate different output for different browsers.
 
 * In general, ES6 is faster than the ES5 equivalent, so try to serve ES6 to browsers that support it.
 * TypeScript has slightly buggy template literal support when compiling to ES5, which can hurt performance.
```

Comment:
Can we update this too? There seems to be no actionable advice to the TypeScript section. What we really recommend is targeting ES2017/modules from tsc and doing any other transforms in Rollup/Babel.

TypeScript's issue is non-spec-compliance when compiling to ES5 and a module setting of something other than  `esnext`, `es2015`, etc. And it's not just slightly buggy, it completely breaks lit-html's ability to detect a re-render, so all DOM is recreated all the time. So we should be very adamant here that users not compile their code to ES5/non-modules with TypeScript.

Reply:
Oh, super helpful. Yeah, I'm not sure we captured that in the main LitElement build docs, either. Will update here then check.